common/db_test.go

```diff
@@ -578,3 +582,311 @@ func TestDBUnbondingLocks(t *testing.T) {
 		return
 	}
 }
+
+func TestInsertWinningTicket_GivenValidInputs_InsertsOneRowCorrectly(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	sessionID, ticket, sig, recipientRand := defaultWinningTicket(t)
+
+	err = dbh.StoreWinningTicket(sessionID, ticket, sig, recipientRand)
+
+	if err != nil {
```

Comment:
Could we do the following:

```
if assert.Nilf(err, "failed to insert winning ticket with error: %v", err) {
    ...
    assertions assuming that winning ticket insertion succeeded
    ....
}
```

Reply:
I'm just switching to `require.Nil(err)` for all the instances where we would use `t.Fatalf`